SConstruct

```diff
@@ -467,6 +468,9 @@ if methods.get_cmdline_bool("fast_unsafe", env_base.dev_build):
 if env_base["use_precise_math_checks"]:
     env_base.Append(CPPDEFINES=["PRECISE_MATH_CHECKS"])
 
+if env_base["allow_automatic_updates"]:
+    env_base.Append(CPPDEFINES=["ALLOW_AUTO_UPDATE"])
```

Comment:
Both the option and the define sound like the engine will auto-update, when it just does update checks. So I would rename like this:
```suggestion
if env_base["engine_update_check"]:
    env_base.Append(CPPDEFINES=["ENGINE_UPDATE_CHECK_ENABLED"])
```